src/open_inwoner/templates/pages/cases/document_form.html

```diff
@@ -4,6 +4,6 @@
 {% render_form id="document-upload" form=form method="POST" hxencoding="multipart/form-data" hxpost=hxpost_document_action hxtarget="#form_upload" submit_text=_("Bestand uploaden") extra_classes="case-detail-form" %}
     {% csrf_token %}
     {% input form.type no_label=True no_help=True class="label input" id="id_type" extra_classes="file-type__select" %}
```

Comment:
`no_label=True` was already set to True on Develop so you may experience a merge conflict here with the previous commit - All else looking good!